web/src/app/users/UserContactMethodForm.js

```diff
@@ -34,6 +35,24 @@ export default class UserContactMethodForm extends React.PureComponent {
   }
 
   render() {
+    const locale = this.props.value
+
+    let exampleNumber
+
+    try {
```

Comment:
I don't think we need the `try` just have getExampleNumber return a default value